networking_generic_switch/devices/corsa_devices/__init__.py

```diff
@@ -115,10 +115,18 @@ def add_network(self, segmentation_id, network_id, of_controller=None):
             cont_ip, cont_port = of_controller
             if 'controllerNamespace' in self.config:
                 c_controller_namespace = self.config['controllerNamespace']
-  
+
+        isVFCHost = True
+        if not 'VFCHost' in self.config or not self.config['VFCHost'] == 'True':
+            LOG.info("PRUTH: Skipping VFC Creation: isVFCHost " + str(isVFCHost))
```

Comment:
Are these debug statements still useful? Should they at least be logged as `LOG.debug` level instead?